fhir-persistence-schema/README.md

```diff
@@ -275,6 +275,9 @@ For PostgreSQL:
 --db-type postgresql
 ```
 
+Alternatively, you can drop specific schemas with `--drop-schema-batch schema-name-to-drop` and
+`--drop-schema-oauth schema-name-to-drop`
```

Comment:
It seems a little odd to me because we don't allow the user to specify the schema name during the create.

Reply:
```
            case "--create-schema-oauth":
                this.createOauthSchema = true;
                if (nextIdx < args.length && !args[nextIdx].startsWith("--")) {
                    schema.setOauthSchemaName(args[nextIdx]);
                    i++;
                }
                break;

Reply:
if the next value is not a name, but a value it puts it as the oauth schema name, same for fhir, and batch

Reply:
interesting.  i've always just created the schemas with `--create-schemas` which takes the fhir schema name from `--schema-name` property and uses the hardcoded values for the other two.  i do vaguely remember adding the alternative now and listing them under https://github.com/IBM/FHIR/tree/master/fhir-persistence-schema#alternative-setting-up-a-shared-db2-with-separate-schemas-for-each-tenant